Tests/XCUITests/PhotonActionSheetTest.swift

```diff
@@ -34,35 +34,36 @@ class PhotonActionSheetTest: BaseTestCase {
     func testShareOptionIsShown() {
         navigator.openURL(path(forTestPage: "test-mozilla-org.html"))
         waitUntilPageLoad()
-        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.shareButton], timeout: 10)
-        app.buttons[AccessibilityIdentifiers.Toolbar.shareButton].tap()
+        navigator.goto(BrowserTabMenu)
+        waitForExistence(app.tables["Context Menu"].otherElements[ImageIdentifiers.share], timeout: TIMEOUT)
+        navigator.performAction(Action.ShareBrowserTabMenuOption)
 
         // Wait to see the Share options sheet
-        waitForExistence(app.buttons["Copy"], timeout: 10)
+        waitForExistence(app.buttons["Copy"], timeout: TIMEOUT)
     }
 
     // Smoketest
     func testShareOptionIsShownFromShortCut() {
         navigator.openURL(path(forTestPage: "test-mozilla-org.html"))
         waitUntilPageLoad()
-        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.shareButton], timeout: 10)
-        app.buttons[AccessibilityIdentifiers.Toolbar.shareButton].tap()
+        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.settingsMenuButton], timeout: TIMEOUT)
+        navigator.performAction(Action.ShareBrowserTabMenuOption)
 
         // Wait to see the Share options sheet
         if iPad() {
-            waitForExistence(app.buttons["Copy"], timeout: 15)
+            waitForExistence(app.buttons["Copy"], timeout: TIMEOUT)
         } else {
-            waitForExistence(app.buttons["Close"], timeout: 15)
+            waitForExistence(app.buttons["Close"], timeout: TIMEOUT)
         }
     }
 
     func testSendToDeviceFromPageOptionsMenu() {
         // User not logged in
         navigator.openURL(path(forTestPage: "test-mozilla-org.html"))
         waitUntilPageLoad()
-        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.shareButton], timeout: 10)
-        app.buttons[AccessibilityIdentifiers.Toolbar.shareButton].tap()
-        waitForExistence(app.buttons["Send Link to Device"], timeout: 10)
+        navigator.goto(BrowserTabMenu)
+        navigator.performAction(Action.ShareBrowserTabMenuOption)
+        waitForExistence(app.buttons["Send Link to Device"], timeout: TIMEOUT)
```

Comment:
This test started to fail recently. The menu item "Send Link to Device" is no longer in the browser tab menu, but we can first click "Share" and then "Send Link to Device".